src/fw/services/normal/activity/activity_metrics.c

```diff
@@ -677,6 +696,7 @@ void activity_metrics_prv_init(SettingsFile *file, time_t utc_now) {
           new_history.values[new_index] = old_history.values[i];
         }
       }
+      }
```

Comment:
either update indentation here -- or invert the case of it, and just `return` if the RTC is not valid?